Integrations/integration-rasterize.yml

```diff
@@ -56,6 +56,45 @@ script:
             demisto.results(error_message)
         sys.exit(0)
 
+    if demisto.command() == 'rasterize-image':
+        res = demisto.getFilePath(demisto.args()['EntryID'])
+        # demisto.results({'res':res})
+        # res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
+        with open(res['path'], 'r') as f:
+            data = f.read()
+        b64 = base64.b64encode(data)
+        html = '<img src="data:image/png;base64, ' + b64 + '">'
+
+        return_code = 0
+        error_message = ''
+
+        friendlyName = 'image.png'
+        f = open('htmlImage.html','w')
+        f.write('<html style="background:white";>' + html + '</html>')
+        f.close()
+
+        proxy_flag = ""
+        if proxy:
+            proxy_flag = "--proxy=" + http_proxy
+        demisto.debug('rasterize proxy settings: ' + proxy_flag)
+
+        command = ['phantomjs', proxy_flag, '/usr/local/bin/rasterize.js', 'htmlImage.html', friendlyName]
+        if demisto.get(demisto.args(), 'width') and demisto.get(demisto.args(), 'height'):
+            command.append(demisto.gets(demisto.args(), 'width') + '*' + demisto.gets(demisto.args(), 'height'))
+        try:
+            error_message = subprocess.check_output(command)
+        except Exception as e:
+            return_code = -1
+            error_message = e.output
+
+        if return_code == 0:
```

Comment:
I think there is a function in common that does this - `file_result_existing_file`

Reply:
Still does most of it and saves lines, no? Just say `res = file_result_ex.....` and then change the type and return, no?